src/Service/ControllerManagerFactory.php

```diff
@@ -13,7 +13,7 @@
 use Zend\ServiceManager\FactoryInterface;
 use Zend\ServiceManager\ServiceLocatorInterface;
 
-class ControllerLoaderFactory implements FactoryInterface
+class ControllerManagerFactory implements FactoryInterface
```

Comment:
Instead create a new class and inherit the old one from the new one. By this way BC is preserved